src/mudlet-lua/lua/Other.lua

```diff
@@ -700,3 +700,111 @@ function condenseMapLoad()
 
   return loadtime
 end
+
+do
```

Comment:
I do not understand lua enough to understand this construct - when is this `do` done is it once when `Other.lua` is loaded into the profile in Mudlet?

Reply:
In lua the construct `do [...] end` spans a block, scoping local variables within and thus making them "private" to that block.